<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowgl</class>
 <widget class="QMainWindow" name="MainWindowgl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Videowidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlive">
   <property name="text">
    <string>实时预览</string>
   </property>
  </action>
  <action name="actioneq">
   <property name="text">
    <string>录像回放</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Videowidget</class>
   <extends>QOpenGLWidget</extends>
   <header>videowidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
